src/SDKs/PolicyInsights/Management/Management.PolicyInsights/Microsoft.Azure.Management.PolicyInsights.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.PolicyInsights</PackageId>
+    <Description>Provides policy insights operations for Microsoft Azure.</Description>
+    <AssemblyName>Microsoft.Azure.Management.PolicyInsights</AssemblyName>
+    <Version>1.0.0-preview</Version>
+    <PackageTags>Microsoft Azure Policy;Policy Insights;</PackageTags>
```

Comment:
@bulentelmaci package tags should be single word, adding space makes it two tags.
I am trying to get rid of tags that have spaces in it. you can put as many tags you want as long as those tags can be used for customers to search for your nuget package.

Reply:
ok, i'll change those tags then